context/app/static/js/components/workspaces/workspaceMessaging.tsx

```diff
@@ -88,21 +100,35 @@ const text = {
 
 const pageLinks = [
   {
+    onClick: () => trackRelevantPage('Tutorials'),
     link: '/tutorials/workspaces',
     children: 'Tutorials',
   },
   {
+    onClick: () => trackRelevantPage('Templates'),
     link: '/templates',
     children: 'Templates',
   },
   {
+    onClick: () => trackRelevantPage('Dataset Search Page'),
     link: '/search?entity_type[0]=Dataset',
     children: 'Dataset Search Page',
   },
 ];
 
-function LogInAlert() {
-  return <LoginAlert featureName="workspaces" />;
+function WorkspacesLogInAlert() {
+  return (
+    <LoginAlert
+      featureName="workspaces"
+      onClick={() =>
+        trackEvent({
+          category: WorkspacesEventCategories.WorkspaceLandingPage,
+          action: 'Log In / From alert',
+          label: 'alert banner',
+        })
+      }
```

Comment:
We can optionally pull this out into a function like `trackRelevantPage` so it only needs to be instantiated once